release-operator/src/main.rs

```diff
@@ -30,24 +36,43 @@ struct DetectArgs {
     label: String,
 }
 
+#[derive(Args, Debug)]
+struct PublishArgs {
+    /// Token to access crates.io for publishing
+    #[clap(short, long, env = "CARGO_REGISTRY_TOKEN")]
+    token: SecStr,
+
+    /// Repeatable option to provide a list of paths to crates
+    #[clap(short, long = "crate")]
+    crates: Vec<Crate>,
+}
+
 fn main() -> anyhow::Result<()> {
     std::env::set_var(
         "RUST_LOG",
         std::env::var("RUST_LOG").unwrap_or_else(|_| "info".to_string()),
     );
     env_logger::init();
 
+    if log::log_enabled!(log::Level::Trace) {
+        // see https://docs.rs/cmd_lib/latest/cmd_lib/fn.set_debug.html
+        std::env::set_var("CMD_LIB_DEBUG", "1");
+    }
+
     let start = std::time::Instant::now();
     log::trace!("starting release-operator process");
 
     let cli = Cli::parse();
+    log::debug!("got arguments: {cli:#?}");
```

Comment:
Please mind: this operation is safe to to the usage of `secstr::SecStr` which will redact any secrets, like the crates.io token.

Reply:
```suggestion
    // Please mind: this operation is safe due to the usage of `secstr::SecStr`
    // which will redact any secrets, like the crates.io token.
    log::debug!("got arguments: {cli:#?}");
```

:grin: